src/native/libs/System.Native/pal_datetime.m

```diff
@@ -0,0 +1,12 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#include "pal_datetime.h"
+#import <Foundation/Foundation.h>
+
+char* SystemNative_GetDefaultTimeZone()
+{
+    NSTimeZone *tz = [NSTimeZone localTimeZone];
+    NSString *name = [tz name];
+    return (name != nil) ? strdup([name UTF8String]) : strdup("Local");
```

Comment:
Android returns NULL if it can't find the local time:

[https://github.com/dotnet/runtime/blob/7414af2a5f6d8d99efc27d3f5ef7a394e0b23c42/src/native/libs/System.Native/pal_datetime.c#L46-L55](https://github.com/dotnet/runtime/blob/7414af2a5f6d8d99efc27d3f5ef7a394e0b23c42/src/native/libs/System.Native/pal_datetime.c#L46-L55)

do we want to do the same here?

Reply:
Yeah, both should return NULL and fall back to UTC.